packages/fetchai/protocols/aggregation/README.md

```diff
@@ -0,0 +1,42 @@
+# Aggregation Protocol
+
+## Description
+
+This is an aggregation protocol for aggregating observations.
+
+## Specification
+
+```yaml
+---
+name: aggregation
+author: fetchai
+version: 0.1.0
+description: A protocol for agents to aggregate individual observations
+license: Apache-2.0
+aea_version: '>=1.0.0, <2.0.0'
+protocol_specification_id: fetchai/aggregation:0.1.0
+speech_acts:
+  observation:
+    value: pt:int
```

Comment:
Would it not be better to make this more generic and offer a union of permitted types? Or have `value_int: pt:optional[int]` `value_bytes: optional[bytes]`? Otherwise this protocol is quite narrowly usable for only integer type observations.